substrate-node/pallets/pallet-smart-contract/src/lib.rs

```diff
@@ -1797,11 +1797,27 @@ impl<T: Config> PublicIpModifier for Pallet<T> {
 impl<T: Config> Pallet<T> {
     fn is_block_author(signer: &Signer<T, <T as Config>::AuthorityId>) -> Result<(), Error<T>> {
```

Comment:
Maybe renaming the function here will be more accurate

Reply:
Rename to what? I don't think the name is incorrect

Reply:
maybe `is_next_block_author` ?